scene/gui/tree.cpp

```diff
@@ -2624,6 +2624,76 @@ int Tree::draw_item(const Point2i &p_pos, const Point2 &p_draw_ofs, const Size2
 	return htotal;
 }
 
+void Tree::draw_background_stripes() {
+	if (!theme_cache.draw_stripes) {
+		return;
+	}
+
+	const RID ci = get_canvas_item();
+
+	int stripe_count = 0;
+	TreeItem *current_item = root;
+	int total_size = 0;
+
+	// Declaring this variable in the if statement breaks the return value and only ever returns a value of 1.
```

Comment:
What this comment refers to?

Reply:
That's referring to me being really dumb at the time, fixing it 😁